content/_snippets/etl-source-object.md

```diff
@@ -0,0 +1,13 @@
+### ETL Source Object
+<!-- This nested object definition is identical across server_state and server_info -->
+
+On a reporting mode server, each member of the `etl_sources` field is an object with the following fields:
+
+| Field                       | Type    | Description |
+|-----------------------------|---------|-------------|
+| `connected`                 | Boolean | If `true`, the reporting mode server is connected to this p2p mode server. If `false`, that may indicate a network outage or that the p2p mode server is down. |
```

Comment:
`false` indicates the reporting server can't connect to the p2p server for whatever reason. Could be down, the port could be specified incorrectly, could be getting rate limited, etc.